advisories/a82576.md

```diff
@@ -0,0 +1,206 @@
+---
+title: Technical Advisory 82576
+advisory: A-82576
+summary: Adding a column to a table which references a sequence, or creating a table with columns referencing sequences, adds an incomplete back-reference to the column to the sequence metadata.
```

Comment:
Minor comment: I'm concerned that this repetition of "to" in the last clause might be confusing. To be clear, the back-reference exists within the sequence metadata, and refers to both the column and the table the column is in.

Reply:
would it make more sense to just say "...adds an incomplete back-reference to the sequence metadata."

Reply:
Yes, absolutely.